ssl/statem/statem_srvr.c

```diff
@@ -965,6 +965,17 @@ WORK_STATE ossl_statem_server_post_work(SSL *s, WORK_STATE wst)
                         SSL3_CC_APPLICATION | SSL3_CHANGE_CIPHER_SERVER_WRITE))
             /* SSLfatal() already called */
             return WORK_ERROR;
+
+#ifndef OPENSSL_NO_QUIC
+            if (SSL_IS_QUIC(s)
+                && s->ext.early_data == SSL_EARLY_DATA_ACCEPTED) {
```

Comment:
Nit: This should fit on a single line. Generally, if an `if()` extends over multiple lines, I prefer (and I think upstream does, too), an extra indentation to keep the conditional separate from the code block. Kinda like what you did on lines 973 and 974 just below this.